### DISCUSSION TITLE
Panel refresh for OLED monitors

### DISCUSSION THREAD

**Discussion by backslashV** [rockowitz/ddcutil#475] [Ideas]
I was wondering if it's possible to add the option to manually trigger a panel refresh for OLED monitors.

**Comment by rockowitz** [comment]
ddcutil can only communicate with monitors using DDC/CI, and obviously there's no VCP feature defined in the Monitor Control Command set to trigger panel refresh.  It's conceivable, though highly unlikely, is that a command to refresh the panel is communicated using an otherwise undefined VCP feature, in which case you could write a User Defined Feature file for your monitor.
